client/components/plans/plan-actions/index.jsx

```diff
@@ -246,6 +254,7 @@ module.exports = React.createClass( {
 			<div>
 				<span className="plan-actions__current-plan-label" onClick={ this.recordCurrentPlanClick }>
 				{ this.translate( 'Your current plan', { context: 'Informing the user of their current plan on /plans/' } ) }</span>
+				{ this.freePlanExpiration() }
```

Comment:
At least for me it seems to say "Never expires" on the plans page when you haven't yet started a trial